packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -73,10 +74,15 @@ function LoginForm({
         name="identifier"
         label="Email"
         type="email"
-        value={email}
+        defaultValue={email}
```

Comment:
I think you still need value as well right to sync the change?

Reply:
good catch, I was testing something when I changed this, rolled back now